doc/connectivity/networking/native_sim_setup.rst

```diff
@@ -145,18 +145,28 @@ For TCP test, type:
 
    ./echo-client -t 127.0.0.1
 
-Setting interface name from command line
-****************************************
+Setting interface name and IPv4 parameters from command line
+************************************************************
 
 By default the Ethernet interface name used by native_sim is determined by
 :kconfig:option:`CONFIG_ETH_NATIVE_POSIX_DRV_NAME`, but is also possible
 to set it from the command line using ``--eth-if=<interface_name>``.
+
+The same applies to the IPv4 address, gateway and netmask. Note that the values
+can only be set when respectively
```

Comment:
That's not really true as I've explained above. Perhaps we could mention that it'd make sense to disable those configs, but it's not really a deal breaker.